lxd/migrate_container.go

```diff
@@ -150,7 +150,7 @@ func (s *migrationSourceWs) checkForPreDumpSupport() (bool, int) {
 	if tmp != "" {
 		use_pre_dumps = shared.IsTrue(tmp)
 	}
-	logger.Debugf("migration.incremental.memory %d", use_pre_dumps)
+	logger.Debugf("migration.incremental.memory %t", use_pre_dumps)
```

Comment:
Looks like temporary debug statement that should have been dropped, no?

Reply:
It's `Debugf()` so they are usually never seen be users and it helps debugging. :)

Reply:
But probably fine to remove it too